pkg/platform/kube/platform.go

```diff
@@ -472,6 +472,11 @@ func (p *Platform) EnrichFunctionConfig(ctx context.Context, functionConfig *fun
 		functionConfig.Spec.ServiceAccount = p.Config.Kube.DefaultFunctionServiceAccount
 	}
 
+	// enrich foo
+	if functionConfig.Spec.Foo == nil {
+		functionConfig.Spec.Foo = p.Config.Foo
+	}
```

Comment:
This is already done in the abstract platform, which is called above in line 428:
```
if err := p.Platform.EnrichFunctionConfig(ctx, functionConfig); err != nil {
		return err
	}
```

So no need to do it twice.